vms/platformvm/txs/executor/standard_tx_executor.go

```diff
@@ -982,37 +975,33 @@ func (e *StandardTxExecutor) SetSubnetValidatorWeightTx(tx *txs.SetSubnetValidat
 	}
 
 	// Verify the flowcheck
-	fee, err := e.FeeCalculator.CalculateFee(tx)
+	fee, err := e.feeCalculator.CalculateFee(tx)
 	if err != nil {
 		return err
 	}
 
-	if err := e.Backend.FlowChecker.VerifySpend(
+	if err := e.backend.FlowChecker.VerifySpend(
 		tx,
-		e.State,
+		e.state,
 		tx.Ins,
 		tx.Outs,
-		e.Tx.Creds,
+		e.tx.Creds,
 		map[ids.ID]uint64{
-			e.Ctx.AVAXAssetID: fee,
+			e.backend.Ctx.AVAXAssetID: fee,
 		},
 	); err != nil {
 		return err
 	}
 
+	// Parse the warp message.
 	warpMessage, err := warp.ParseMessage(tx.Message)
 	if err != nil {
 		return err
 	}
-	if warpMessage.NetworkID != e.Ctx.NetworkID {
-		return fmt.Errorf("expected networkID %d but got %d", e.Ctx.NetworkID, warpMessage.NetworkID)
-	}
```

Comment:
why we don't need this anymore?

Reply:
This PR does not include warp signature verification. This is added in #3423